src/sass/table/table.scss

```diff
@@ -0,0 +1,86 @@
+@import "../variables/variables";
+
+:root {
+    /* these exclude the 16px padding for top/bottom */
+    --density-compact-cell-height: 36px;
+    --density-default-cell-height: 48px;
+    --density-relaxed-cell-height: 56px;
+}
+
+.table {
+    overflow-x: auto;
+    position: relative;
+}
+
+.table table {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+}
+
+// default styles here, caption showing up top
+.table caption {
+    text-align: left;
+}
+
+.table--caption-bottom caption {
+    caption-side: bottom;
+}
+
+.table--caption-clipped caption {
+    border: 0;
+    clip: rect(1px, 1px, 1px, 1px);
+    height: 1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    white-space: nowrap;
+    width: 1px;
+}
+
+.table th,
+.table td {
+    // explicit background is needed for sticky columns
+    background-color: var(--color-background-primary);
+    border-bottom: 1px solid var(--color-stroke-subtle);
+    box-sizing: border-box;
+    height: var(--density-default-cell-height);
+    max-width: 400px;
+    min-width: 124px;
+
+    // minimal padding to prevent text from touching the border
+    padding-block: var(--spacing-100);
+    padding-inline: var(--spacing-200);
+}
+
+.table .table__cell {
```

Comment:
Minor, but we should probably reduce the amount of nesting in these selectors if specificity isn't needed, e.g. `.table .table__cell` becomes just `.table__cell`

Reply:
Good catch.